/**:
  ros__parameters:
    pedestrian_simulator:
      static: false
      node:
        debug_output: false
        pretend_to_be_optitrack: false
        scenario: random_social/8_corridor.xml
        update_frequency: 20.0
        prediction_step: 0.2
        horizon: 50
        use_path_origin: false
      pedestrians:
        seed: 5 # > 0,  -1 = random!
        single_scenario: -1 # > 0 = run the scenario with this index | -1 = not a single scenario
        type: gaussian # Only used if it is not in the scenario file
        # process_noise: [.8, .8] # Excessive
        process_noise: [.3, .3] # Safe-Horizon
        # process_noise: [0.0, 0.0] # Deterministic
        collision_free_spawn: true
        interaction: true
        constant_velocity_predictions: true # If "false" will produce predictions that follow the pedestrian model
      binomial:
        p: 0.025
      social_forces:
        strength: 10.
        decay: 1.07
        line_of_sight: 0.7
